src/ActionModal/components/SendModal.vue

```diff
@@ -340,6 +345,27 @@ export default {
         }
       }
     },
+    denom: {
+      query: gql`
+        query NetworksDelegationModal($networkId: String!) {
+          network(id: $networkId) {
+            id
+            stakingDenom
+          }
+        }
+      `,
+      fetchPolicy: "cache-first",
+      /* istanbul ignore next */
+      variables() {
+        return {
+          networkId: this.network
+        }
+      },
+      /* istanbul ignore next */
+      update(data) {
+        return data.network.stakingDenom
```

Comment:
```suggestion
        if (data.network) return data.network.stakingDenom
        return ''
```

Reply:
:thinking: then I should apply this to the other modals too...

Shouldn't this be solved once we add the `error` field in all apollo queries?